.github/workflows/release_builds.yml

```diff
@@ -0,0 +1,51 @@
+name: Releases
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+-sumo-[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-sumo-[0-9]+-alpha.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-sumo-[0-9]+-beta.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-sumo-[0-9]+-rc.[0-9]+'
+env:
+  SIDECAR_IMAGE: "ghcr.io/sumologic/tailing-sidecar"
+  OPERATOR_IMAGE: "ghcr.io/sumologic/tailing-sidecar-operator"
+  LATEST_TAG: "latest"
+
+jobs:
+  build:
```

Comment:
🤔 Maybe we can separate operator and sidecar build steps into 2 separate jobs? They'd then run in parallel (=faster) and finding what went wrong would be easier.

What do you think?